app/routes.py

```
@@ -125,11 +137,13 @@
 from scripts.update_chrome_shortcut import (
     update_chrome_shortcuts_info,
     shortcuts_need_patch,
+    first_shortcut_path,
 )
 from app.utils.screenshots import (
     is_chrome_debug_port_open,
```

Ruff (0.11.9)

Redefinition of unused `is_chrome_debug_port_open` from line 96

Remove definition: `is_chrome_debug_port_open`

(F811)